lib/phlex/rails/rendering.rb

```diff
@@ -0,0 +1,11 @@
+module Phlex
+  module Rails
+    module Rendering
+      def render(...)
+        _raw ApplicationController.render(...)
+      end
+    end
+  end
+end
+
+Phlex::Context.send(:include, Phlex::Rails::Rendering)
```

Comment:
Do we need to `send` this or can we just call `include` directly?

```suggestion
Phlex::Context.include(Phlex::Rails::Rendering)
```